app/helpers/product_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module ProductHelper
+  def format_price(currency, price)
+    "#{currency.upcase} #{price.to_f / 100}"
```

Comment:
Why are we dividing there?

Reply:
Is that a common practice? Maybe change the field type to float?

Reply:
This is how Stripe prices worked

Reply:
Alright. It makes sense in terms of discounts, because we can avoid weird fractions when giving a percentage discount.